RPG-game-Hung/Readme.md

```diff
@@ -0,0 +1,15 @@
+This game is a classical fight game! It can be an old fashioned slap fight, or use magic or item
+
+You can choose number of players against the computer (maximum 5)
+
+There are 2 play modes of the game:
+    1. You and computer will fight against computers OR
+    2. You will control all your team and fight against computers
+
+Beware: computers might be geniuses, or dummies depends on the mood, or the weather!
+
+Run file main.py to play, package termcolor is required
+
+Have fun!
+
+Hung - 15.03.2020
```

Comment:
Also, `Readme.md` should include instruction on how to install/run the program. For example, something like this:  
## How to run
- Create a virtual environment with `virtualenv` or simply run `python -m venv venv`
- Activate the virtual environment, and run `pip install -r requirements.txt`
- run `python main.py`